packages/apps/public/index.html

```diff
@@ -6,6 +6,7 @@
     <link rel="manifest" href="manifest.json">
     <link rel="shortcut icon" href="favicon.ico">
     <title><%= htmlWebpackPlugin.options.PAGE_TITLE %></title>
+    <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,400;0,800;1,200;1,400&display=swap" rel="stylesheet">
```

Comment:
Really would prefer not pulling additional fonts from google. See this issue as well https://github.com/polkadot-js/apps/issues/3161 (which is WIP, a big driver behind removing semantic-ui). 

If we want a consistent font, need one with the correct licenses added specifically to the repo. (Really trying to get to self-contained without relying on centralized services)